_pytest/assertion/rewrite.py

```diff
@@ -35,6 +35,12 @@
 REWRITE_NEWLINES = sys.version_info[:2] != (2, 7) and sys.version_info < (3, 2)
 ASCII_IS_DEFAULT_ENCODING = sys.version_info[0] < 3
 
+if sys.version_info >= (3,5):
+    ast_Call = ast.Call
+else :
+    ast_Call = lambda a,b,c : ast.Call(a, b, c, None, None)
```

Comment:
PEP-8 says no spaces in front of ":".

It's the `else :` that really leaps out to me (nails-on-blackboard style); I wouldn't have complained just about the lambda.